### PR TITLE
chore: Upgrade libraries versions.

### PR DESCRIPTION
**Pull request by corentin-stamper**
## Changes description 🧑‍💻

| library | old version | new version |
|---|---|---|
|**Android Gradle plugin**| 7.2.1 | 7.3.1 |
|**Kotlin Gradle plugin**| 1.6.21 | 1.7.0 |
|**sonarqube**| 3.3 | 3.5.0.2730 |
|**ktlint**| 10.2.0 | 11.0.0 |
|**vitamin-android**| 0.4.0-beta01 | 0.4.0 |
|**refreshVersions**| 0.40.2 | 0.51.0 |
|**customview-poolingcontainer**| 1.0.0-beta02 | 1.0.0 |
|**compose**| 1.2.0-rc01 | 1.2.0 |
|**accompanist**| 0.24.11-rc | 0.25.0 |
|**espresso**| 3.4.0 | 3.5.0 |
|**junit**| 1.1.3 | 1.1.4 |

## Checklist ✅

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check compose contract convention. It must follow conventions described [here](/CONVENTIONS.md).
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on a phone device/emulator.
- I have tested on a tablet device/emulator.
- I have tested on a large screen device/emulator.
- If it includes design changes, please ask for a review with a core team designer.

### CODE REVIEW

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/e5d6caf8-7ad4-4891-adac-7fc6b926dabb/artifact/9b3f7e935b2a01cd/p/afd790205a7f1f80ae994cb7ff940d11

File: settings.gradle.kts [hunk 1]
```diff
@@ -35,4 +35,4 @@ include(":menus")
 include(":snackbars")
 include(":badges")
 include(":quantity-pickers")
-include(":vitamin")
+// include(":vitamin")
```
Comment (corentin-stamper) [inline settings.gradle.kts:38]: Build issue. Need more analysis.

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/3424910f-4066-41c4-ba9f-4e531047407c/artifact/3759476721d07b85/p/722ebaed829d39f6b6f813159e1f2b34

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/bb9034f3-1f12-470f-bd70-130ccb2effb8/artifact/27be1608bd92b660/p/1f8601cd99ecae6df1de4574d8b37e79

**Review by GerardPaligot — APPROVED** [review]

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/aeec70ac-55e1-439d-9f05-deeb1949e317/artifact/a492e1387c8eba4e/p/a665c48b85f86c633517d962271e7df9

**Pull request merged**